<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsView</class>
 <widget class="QDialog" name="SettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>611</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>329</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nazwa firmy:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="companyNameInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Adres firmy:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="companyAddressInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>NIP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nipInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Host SMTP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="smtpHostInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Port SMTP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="smtpPortInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Użytkownik SMTP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="smtpUserInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Hasło SMTP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="smtpPassInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Nazwa nadawcy email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailNameInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Adres email nadawcy:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailAddressInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Stopka dokumentu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="footerInput"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
